packages/flutter/lib/src/material/switch.dart

```diff
@@ -229,6 +232,11 @@ class Switch extends StatefulWidget {
   /// The color for the button's [Material] when a pointer is hovering over it.
   final Color? hoverColor;
 
+  /// The splash radius.
```

Comment:
I don't know what we should update this too but I definitelly feel like we should change it

Reply:
Updated now.